### PR TITLE
Nifi-5171 - fixed Yandex Jersey issues by adding dependency to POM and modified A…

### PR DESCRIPTION
**Pull request by veteranbv**
…PI call to now detect for languages

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [x] Have you written or updated unit tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [x] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [x] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [x] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/pom.xml [hunk 1]
```diff
@@ -53,7 +53,12 @@
             <artifactId>jersey-media-json-jackson</artifactId>
             <version>${jersey.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
```
Comment (mattyb149) [inline nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/pom.xml]: I think this is licensed under CDDL 1.1? If so it should be included in the NAR's NOTICE file along with the other Jersey stuff.

Comment (veteranbv) [reply nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/pom.xml]: I'll get the following line added to the NOTICE file:
`    (CDDL 1.1) (GPL2 w/ CPE) jersey-hk2 (org.glassfish.jersey.inject:jersey-hk2:jar:2.26 - https://jersey.github.io/)`

Comment (veteranbv) [reply nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/pom.xml]: @mattyb149 Added

File: nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/src/main/java/org/apache/nifi/processors/yandex/YandexTranslate.java [hunk 2]
```diff
@@ -211,10 +211,14 @@ public void destroyClient() {
     protected Invocation prepareResource(final String key, final List<String> text, final String sourceLanguage, final String destLanguage) {
         Invocation.Builder builder = client.target(URL).request(MediaType.APPLICATION_JSON);
 
-        final MultivaluedHashMap entity = new MultivaluedHashMap();;
+        final MultivaluedHashMap entity = new MultivaluedHashMap();
         entity.put("text", text);
         entity.add("key", key);
-        entity.add("lang", sourceLanguage + "-" + destLanguage);
+        if (sourceLanguage != "") {
```
[the remark this answers is not in the document]
Comment (veteranbv) [reply nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/src/main/java/org/apache/nifi/processors/yandex/YandexTranslate.java]: @joewitt Updated using your recommendations

File: nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/src/main/java/org/apache/nifi/processors/yandex/util/Languages.java [hunk 3]
```diff
@@ -25,6 +25,7 @@
     private static final Map<String, String> languageAbbreviationMap = new HashMap<>();
 
     static {
+        languageAbbreviationMap.put("", "blank");
```
[the remark this answers is not in the document]
Comment (veteranbv) [reply nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/src/main/java/org/apache/nifi/processors/yandex/util/Languages.java]: @joewitt I took a little different route by deleting the blank reference in this file and updating the validator logic

**Pull request closed without merging**